src/platform/site-wide/header/components/App/index.js

```diff
@@ -72,11 +71,6 @@ export const App = ({
 
     const onResize = debounce(100, deriveIsDesktop);
 
-    recordEvent({
```

Comment:
This event was [added 3 years ago](https://github.com/department-of-veterans-affairs/vets-website/pull/19574). [Here's the ticket for it](https://github.com/department-of-veterans-affairs/va.gov-team/issues/33508). Given that the ticket refers to this as a "new product release" and it's been 3 years, we're safe to remove it. Confirmed with MichelleM and Jamie from the Analytics team. [Slack thread here](https://dsva.slack.com/archives/C52CL1PKQ/p1730989615670319)